devito/symbolics/printer.py

```diff
@@ -255,6 +274,11 @@ def _print_Fallback(self, expr):
     _print_Basic = _print_Fallback
 
 
+# Lifted from SymPy so that we go through our own `_print_math_func`
+for k in ('exp log sin cos tan ceiling floor').split():
```

Comment:
Isn't it ceil not ceiling

Reply:
somehow it gets remapped internally...

https://github.com/sympy/sympy/blob/master/sympy/printing/c.py#L734